src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -0,0 +1,313 @@
+      "accountId": "my_new_relic_account_id"
+    },
+
+    "git": {
+      "username": "my_git_access_token"
+    }
+  }
+}
+```
+
+</Step>
+
+<Step>
+
+## Let's run this thing!
+Now that everything is setup, you can run the deployer in the docker image. Because this is actually provisioning infrastructure, deploying apps, etc. it will take a few minutes. Go ahead and invoke the command and then we'll walk through what's happening.
+
+```bash
+docker run -it\
+    -v $HOME/configs/:/mnt/deployer/configs/\
+    --entrypoint ruby deployer main.rb -c configs/[user].docker.local.json -d documentation/tutorial/user_stories/Hello/hello.json
+```
+
+## hello.json
+The deployer is configuration driven, and [hello.json](https://github.com/newrelic/demo-deployer/blob/main/documentation/tutorial/user_stories/Hello/hello.json) is the default configuration for this example. Looking at this file, you will notice several interesting sections. 
+* services
+* global_tags
+* resources
+* instrumentation
+These section describe, to the deployer, the actions to take when it executes. For example, "global_tags" will, as the name implies, apply these tags to ALL the resources in your stack. This is how we apply the consistent tagging strategy discussed above. 
+
+```json
+{
+  "services": [
+    {
+      "id": "app1",
+      "display_name": "Hello-App1",
+      "source_repository": "-b main git@github.com:newrelic/demo-nodetron.git",
+      "deploy_script_path": "deploy/linux/roles",
+      "port": 5001,
+      "destinations": ["host1"],
+      "files":[{
+        "destination_filepath": "engine/data/index.json",
+        "content": [
+          "<h1>Welcome!</h1>",
+          "Congrats! You have successfully setup our <b>Hello World</b> demo environment. So, what's been deployed?",
+          "<ul><li>A simple host on your cloud provider, e.g. AWS, Azure.</li>",
+          "<li>A simple node application that is serving up this page.</li>",
+          "<li>New Relic instrumentation, specifically, the infrastructure agent and the node agent.</li></ul>",
+          " ",
+          "Next, start generating some traffic on the app. This can be done by clicking the <b>Inventory</b> button, which will simply return a collection of items. You can then click each item to see its details.",
+          "You can also use postman, or a similar tool to generate a burst of requests. The APIs for the app are quite simple:",
+          "<ul><li>Inventory <a href=\"[service:app1]/api/inventory\" target=\"_blank\">[service:app1]/api/inventory</a></li>",
+          "<li>Item details <a href=\"[service:app1]/api/inventory/1\" target=\"_blank\">[service:app1]/api/inventory/[item number]</a></li></ul>",
+          " ",
+          "Now, head on over to New Relic and explore what's been created.",
+          "<ul><li>Search for your app and host using [global search](https://one.newrelic.com/redirect/search). Try searching on one of the global tags in your configuration!</li>",
```

Comment:
I just made an update to the hello.json, you might update this line and the next one, the links were not showing correctly once Nodetron was deployed and accessed.